src/Mealz/MealBundle/Controller/ParticipantController.php

```diff
@@ -188,11 +165,10 @@ public function list(DayRepository $dayRepo): Response
     }
 
     /**
-     * edit participation
      * @param Week $week
```

Comment:
Redundant @param and @return annotations.

Reply:
Removed.